MacrobenchmarkSample/app/proguard-rules.pro

```diff
@@ -18,4 +18,11 @@
 
 # If you keep the line number information, uncomment this to
 # hide the original source file name.
-#-renamesourcefileattribute SourceFile
\ No newline at end of file
+#-renamesourcefileattribute SourceFile
+
+# Turn off obfuscations for Baseline Profile Generation
+# This is because Baseline Profile rules participate in the build and will eventually get
```

Comment:
Suggest rewording:

Baseline profiles must be generated from an unobfuscated build, so that the text format matches source. Subsequently, when compiled into binary format within your release APK, they will account R8 obfuscation mapping.